src/SFA.DAS.QnA.Api.Views/Views/QuestionPartials/_TabularData.cshtml

```diff
@@ -1,10 +1,11 @@
-@using Newtonsoft.Json
+@using System.Text.Json
 @model dynamic
 
 @{
     var isError = Model.ErrorMessages != null && Model.ErrorMessages.Count > 0 ? "govuk-input--error" : "";
 
-    var tableData = JsonConvert.DeserializeObject<dynamic>(Model.Value);
+    //var tableData = JsonSerializer.Deserialize<dynamic>(Model.Value);
```

Comment:
Commented out code here and in the line below you have spelt ModelValueDeserializer different to ModelValueDeserialiser - so this would just blow up at run time

Reply:
This isn't done yet.

Reply:
Had a chat with Chris and it turns out that some parts of the code need rewriting before we can safely change the deserialisation in this file. Please revert all changes to this file. I've updated the ticket accordingly